src/TTrigger.h

```diff
@@ -181,6 +184,8 @@ class TTrigger : public Tree<TTrigger>
     QColor mBgColor;
     bool mIsColorizerTrigger;
     bool mModuleMember;
+    // -1: don't self-destruct, 0: delete, 1+: number of times it can still match
```

Comment:
Adjust verb: "number of times, trigger can still fire", etc.
Compare comment in TTrigger.cpp line 1013

Reply:
Blimey @Kebap are you getting as picky about grammar as I *sometimes* seem to be? After all it is just a comment... :stuck_out_tongue_closed_eyes:

Reply:
No complaints here from me, I take in all your grammar improvements!